src/components/Chat/ChatInputComponent.tsx

```diff
@@ -28,6 +28,23 @@ const ChatInputComponent = (props: IProps) => {
     }
   };
 
+  const onAddUser = event => {
+    event.preventDefault();
+    const socketMessage = createJoinMessage(
+      message,
+      roomID,
+    );
+
+    socketSend(socketMessage);
+  }
+
+  const getAvailableChat = event => {
```

Comment:
```suggestion
  const getAvailableChat: void = (event: MouseEvent) => {
```
Alltid greit å ha med typer! :)